tools/qemu_args.sh

```diff
@@ -50,8 +50,9 @@ if [ "$1" = "iommu" ]; then
     IOMMU_DEV_EXTRA=",iommu_platform=on,ats=on"
     IOMMU_EXTRA_ARGS="\
         -device intel-iommu,intremap=on,device-iotlb=on \
-        -device ioh3420,id=pcie.0,chassis=1\
+        -device ioh3420,id=pcie.0,chassis=1 \
     "
+    # TODO: Add support for enabling IOMMU on AMD platforms
```

Comment:
I am wondering if you are suggesting that there's issues with this script running on AMD machines; or if our kernel don't support AMD-IOMMU?

Reply:
Yes, we need to further verify whether our kernel functions correctly with IOMMU enabled on AMD machines. Basically the qemu command `-device intel-iommu` in the script should be changed to `-device amd-iommu`.